### ISSUE TITLE
Error feedback

### ISSUE THREAD

**Issue by 5igm** [Arckenver/Nations#15]
This is another one. It showed after I restarted server.

forge-1.8.9-11.15.1.1902-1.8.9-universal
spongeforge-1.8.9-1890-4.2.0-BETA-1653
Nations-0.10

```
[03:55:07] [Server thread/ERROR] [Sponge/sponge]: Could not pass FMLServerStartedEvent to org.spongepowered.mod.plugin.SpongeModPluginContainer@329005ff
java.util.NoSuchElementException: No value present
    at java.util.Optional.get(Unknown Source) ~[?:1.8.0_101]
    at com.arckenver.nations.serializer.NationDeserializer.deserialize(NationDeserializer.java:71) ~[NationDeserializer.class:0.10]
    at com.arckenver.nations.serializer.NationDeserializer.deserialize(NationDeserializer.java:20) ~[NationDeserializer.class:0.10]
    at com.google.gson.TreeTypeAdapter.read(TreeTypeAdapter.java:58) ~[TreeTypeAdapter.class:?]
    at com.google.gson.Gson.fromJson(Gson.java:803) ~[Gson.class:?]
    at com.google.gson.Gson.fromJson(Gson.java:768) ~[Gson.class:?]
    at com.google.gson.Gson.fromJson(Gson.java:717) ~[Gson.class:?]
    at com.google.gson.Gson.fromJson(Gson.java:689) ~[Gson.class:?]
    at com.arckenver.nations.DataHandler.load(DataHandler.java:68) ~[DataHandler.class:0.10]
    at com.arckenver.nations.NationsPlugin.onStart(NationsPlugin.java:135) ~[NationsPlugin.class:0.10]
    at org.spongepowered.common.event.listener.GameStartedServerEventListener_NationsPlugin_onStart12.handle(Unknown Source) ~[?:?]
    at org.spongepowered.common.event.RegisteredListener.handle(RegisteredListener.java:95) ~[RegisteredListener.class:1.8.9-1890-4.2.0-BETA-1653]
    at org.spongepowered.mod.event.SpongeModEventManager.post(SpongeModEventManager.java:201) [SpongeModEventManager.class:1.8.9-1890-4.2.0-BETA-1653]
    at org.spongepowered.mod.event.SpongeModEventManager.post(SpongeModEventManager.java:242) [SpongeModEventManager.class:1.8.9-1890-4.2.0-BETA-1653]
    at org.spongepowered.mod.SpongeMod.onStateEvent(SpongeMod.java:185) [SpongeMod.class:1.8.9-1890-4.2.0-BETA-1653]
    at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method) ~[?:1.8.0_101]
    at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source) ~[?:1.8.0_101]
    at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source) ~[?:1.8.0_101]
    at java.lang.reflect.Method.invoke(Unknown Source) ~[?:1.8.0_101]
    at com.google.common.eventbus.EventSubscriber.handleEvent(EventSubscriber.java:74) [minecraft_server.1.8.9.jar:?]
    at com.google.common.eventbus.SynchronizedEventSubscriber.handleEvent(SynchronizedEventSubscriber.java:47) [minecraft_server.1.8.9.jar:?]
    at com.google.common.eventbus.EventBus.dispatch(EventBus.java:322) [minecraft_server.1.8.9.jar:?]
    at com.google.common.eventbus.EventBus.dispatchQueuedEvents(EventBus.java:304) [minecraft_server.1.8.9.jar:?]
    at com.google.common.eventbus.EventBus.post(EventBus.java:275) [minecraft_server.1.8.9.jar:?]
    at net.minecraftforge.fml.common.LoadController.sendEventToModContainer(LoadController.java:211) [LoadController.class:?]
    at net.minecraftforge.fml.common.LoadController.propogateStateMessage(LoadController.java:189) [LoadController.class:?]
    at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method) ~[?:1.8.0_101]
    at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source) ~[?:1.8.0_101]
    at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source) ~[?:1.8.0_101]
    at java.lang.reflect.Method.invoke(Unknown Source) ~[?:1.8.0_101]
    at com.google.common.eventbus.EventSubscriber.handleEvent(EventSubscriber.java:74) [minecraft_server.1.8.9.jar:?]
    at com.google.common.eventbus.SynchronizedEventSubscriber.handleEvent(SynchronizedEventSubscriber.java:47) [minecraft_server.1.8.9.jar:?]
    at com.google.common.eventbus.EventBus.dispatch(EventBus.java:322) [minecraft_server.1.8.9.jar:?]
    at com.google.common.eventbus.EventBus.dispatchQueuedEvents(EventBus.java:304) [minecraft_server.1.8.9.jar:?]
    at com.google.common.eventbus.EventBus.post(EventBus.java:275) [minecraft_server.1.8.9.jar:?]
    at net.minecraftforge.fml.common.LoadController.redirect$onPost$0(LoadController.java:53) [LoadController.class:?]
    at net.minecraftforge.fml.common.LoadController.distributeStateMessage(LoadController.java:118) [LoadController.class:?]
    at net.minecraftforge.fml.common.Loader.serverStarted(Loader.java:800) [Loader.class:?]
    at net.minecraftforge.fml.common.FMLCommonHandler.handleServerStarted(FMLCommonHandler.java:289) [FMLCommonHandler.class:?]
    at net.minecraft.server.MinecraftServer.run(MinecraftServer.java:443) [MinecraftServer.class:?]
    at java.lang.Thread.run(Unknown Source) [?:1.8.0_101]
```

**Comment by Arckenver** [comment]
This error is caused by Nations trying to get a world that don't exist anymore. Did you recently delete a world ?

_N.B. when pasting error stacktrace, write ``` before and after the stacktrace so it'll look better_

**Comment by 5igm** [comment]
Yes. I am testing some features of other plugins. Thank you very much.

**Comment by Arckenver** [comment]
Yeah so that's what I thought it was. I'll solve this soon.

Thanks for reporting !

**Comment by Arckenver** [comment]
Fixed as of commit 86e3c2ba9e82c84431f2e5c774c79535a889c314

**Comment by 5igm** [comment]
Thank you very much.
